openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -88,6 +88,7 @@ def to_representation(self, user):
             "level_of_education": AccountLegacyProfileSerializer.convert_empty_to_None(profile.level_of_education),
             "mailing_address": profile.mailing_address,
             "requires_parental_consent": profile.requires_parental_consent(),
+            "account_privacy": UserPreference.get_value(user, 'account_privacy'),
```

Comment:
Hi @wajeeha-khalid! @cahrens is referring to our edX documentation (docs.edx.org) We have API documentation, including for the user preferences API, but it might need to be updated. Here is a link to the latest published version. 
http://edx.readthedocs.org/projects/edx-platform-api/en/latest/